third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -625,6 +635,9 @@ where
                         .pop_as::<Closure>()
                         .map_err(|e| set_err_info!(current_frame, e))?
                         .unpack();
+
+                    let fn_guard = VM_PROFILER.function(&fun);
```

Comment:
For consistency, we should call this after `as_resolved` to not measure loading time, like we do for calls (few lines below)